server/controllers/review.js

```diff
@@ -0,0 +1,44 @@
+const Review = require('../models/Review');
+const asyncHandler = require("express-async-handler");
+
+// @route GET /review
+// @desc Get all reviews of a user
+// @access Public
+exports.userReviews = asyncHandler(async (req, res, next) => {
+  const { uid } = req.params;
+
+  const reviews = await Review.find({ userId: uid });
+
+  if (!reviews) {
+    res.status(404);
+    throw new Error('No reviews found for this user.');
```

Comment:
Express-async-handler will handle the JSON response for errors thrown in our controllers. This is okay 😄

Reply:
Good though, the status is returned, perhaps returning it with a message instead of throwing and error would make for better controller best practices :)

Reply:
Updated this one